test/unit/DCAStrategies/dca-strategies-management-handler.spec.ts

```diff
@@ -339,4 +352,18 @@ contract('DCAStrategiesManagementHandler', () => {
       expect(s.token).to.be.equal(arrayB[i].token);
     });
   }
+
+  function sortTokens(array: IDCAStrategies.ShareOfTokenStruct[]) {
+    for (let i = 0; i < array.length; i++) {
```

Comment:
I think you can sort by a property here, like [this](https://stackoverflow.com/questions/15593850/sort-array-based-on-object-attribute-javascript)

It would be easier to read/understand